hack/local-up-cluster.sh

```diff
@@ -769,20 +769,22 @@ function start_kubelet {
       fi
 
       docker run \
-        --volume=/:/rootfs:ro \
+        --volume=/:/rootfs:ro,rshared \
```

Comment:
OpenShift uses rslave as all mount is done on the host and we had issues with shared mounts - there was a weird mount loop that doubled every mount.